TTS/tts/models/tacotron2.py

```diff
@@ -109,10 +115,26 @@ def __init__(
         # speaker embedding layer
         if self.num_speakers > 1:
             if not self.embeddings_per_sample:
-                speaker_embedding_dim = 512
+                speaker_embedding_dim = 32
```

Comment:
Could this change break compatibility with older models? Perhaps it is interesting to check if the model is multilingual and if so, to change the size of the embedding?

I'm not sure about that, it's something to discuss

Reply:
Of course, I didn't mean to commit that. my bad